runtime/manual/tools/coverage.md

```diff
@@ -0,0 +1,143 @@
+# deno coverage
+
+Print coverage reports from coverage profiles.
+
+## Command
+
+`deno coverage [OPTIONS] <COVERAGE>`
+
+## Synopsis
+
+```bash
+deno coverage [--ignore=<ignore>] [--include=<regex>] [-q|--quiet] [--exclude=<regex>] [--lcov] [--output=<output>] [--html] [--detailed] [-h|--help] <COVERAGE>
+
+deno coverage -h|--help
+```
+
+## Description
+
+Print coverage reports from coverage profiles.
+
+By default, when you run `deno test --coverage` a coverage profile will be generated in the `/coverage` directory of your workspace.
```

Comment:
```suggestion
By default, when you run `deno test --coverage` a coverage profile will be generated in the `/coverage` directory in the current working directory.
```
I think using CWD will be less confusing, WDYT?

Reply:
👍